azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -9,15 +9,15 @@
 
 import logging
 from threading import Event
-from .abstract_clients import AbstractClient, AbstractDeviceClient, AbstractModuleClient
+from .abstract_clients import AbstractClient, AbstractIoTHubDeviceClient, AbstractIoTHubModuleClient
 from .transport import constant
-from .common import Message
+from .models import Message
 from .inbox_manager import InboxManager
 from .sync_inbox import SyncClientInbox
 
 logger = logging.getLogger(__name__)
 
-__all__ = ["DeviceClient", "ModuleClient"]
+__all__ = ["IoTHubDeviceClient", "IoTHubModuleClient"]
 
```

Comment:
is this all and the other one in "init" both needed ?

Reply:
removed

---
In reply to: [275560460](https://github.com/Azure/azure-iot-sdk-python-preview/pull/77#discussion_r275560460) [](ancestors = 275560460)